iOS/testbed/__main__.py

```diff
@@ -448,6 +493,12 @@ def main():
 
     subcommands = parser.add_subparsers(dest="subcommand")
 
+    parser.add_argument(
+        "--catalyst",
+        action="store_true",
+        help="Use Mac Catalyst.",
```

Comment:
A random help message that I realized I could do better.
```suggestion
        help="Run or clone the testbed for a Mac Catalyst build.",
```